openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -711,8 +736,19 @@ else if ( !platform.equals("win32") ) {
 			fileContent += "concurrency " + concurrency + ";\n";
 			fileContent += "timeoutfactor 1" + ";\n";							// All Devtools tests take less than 1h to finish.
 			
-			fileContent += "set jck.env.testPlatform.os \"" + testPlatform + "\";\n";
-			fileContent += "set jck.env.devtools.testExecute.cmdAsFile \"" + pathToJava + "\"" + ";\n";
+			if (platform.equals("win32")) {
+				// On Windows set the testplatform.os to Windows and set systemRoot, but do not
+				// set the file and path separators (an error is thrown if they are set).
+				fileContent += "set jck.env.testPlatform.os \"Windows\";\n";
+			}
+			else {
+				// On other platforms set the testplatform.os to other and set the file and path separators.
+				fileContent += "set jck.env.testPlatform.os \"other\";\n";
+				fileContent += "set jck.env.testPlatform.fileSep \"/\";\n";
+				fileContent += "set jck.env.testPlatform.pathSep \":\";\n";
+			}
```

Comment:
Duplicate code?

Reply:
Which duplication are you referring to?

Reply:
Oh I see - the review Window wasn't expanded properly.
There are three different test suites each of which has a slightly different configuration.
At the moment the code creates the configurations files separately for each test suite - that makes it easy to see what is being written for each suite, but it does lead to some duplication where the parameters are common, as you have observed.  The alternative would be to split of various parts into methods along the lines of 'write the next part' and call those methods inside each of the test suite stanzas so the reader would have to visit those methods to see what is being set.  Unfortunately the parameters are required to appear in a particular order, so I don't think all the common parameters could be added to the configuration in one method.
Swings and roundabouts - as an observer what would you prefer?

Reply:
I'm merging this PR as is because we need to run on AIX and that needs the changes.  We can raise another issue to try to remove the duplicate code if we think it's needed.